aiohttp/web_protocol.py

```diff
@@ -19,8 +19,7 @@
     Union,
     cast,
 )
-
-import attr
+import dataclasses
 import yarl
```

Comment:
Recover the empty line that separates stdlib and third-party sections of imports
```suggestion

import yarl
```